### PR TITLE
GH#32345: Fix example pod configuration

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/openshift-docs/issues/32345

Preview: https://deploy-preview-32871--osdocs.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-using?utm_source=github&utm_campaign=bot_dp#nodes-pods-using-example_nodes-pods-using-ssy

### CODE REVIEW

**Comment by mburke5678** [comment]
@linuxstudio Does this PR meet your expectations in https://github.com/openshift/openshift-docs/issues/32345? Sorry for the delay.

**Comment by mburke5678** [comment]
@sunilcio Can you please take a look at this BZ or assign to someone appropriate? Thank you very much in advance.

**Comment by sunilcio** [comment]
@weinliu could you please help check.

**Comment by weinliu** [comment]
@mburke5678 sorry for being late, /lgtm

**Comment by mburke5678** [comment]
linuxstudio [reviewed in the original issue](https://github.com/openshift/openshift-docs/issues/32345#issuecomment-878585487).

File: modules/nodes-pods-using-example.adoc [hunk 1]
```diff
@@ -9,9 +9,7 @@
 together on one host, and the smallest compute unit that can be defined,
 deployed, and managed.
 
-The following is an example definition of a pod that provides a long-running
-service, which is actually a part of the {product-title} infrastructure: the
-integrated container image registry. It demonstrates many features of pods, most of
+The following is an example definition of a pod from a Rails application. It demonstrates many features of pods, most of
 which are discussed in other topics and thus only briefly mentioned here:
```
Comment (jeana-redhat) [inline modules/nodes-pods-using-example.adoc:13]: Make sense to just unwrap this too?

**Review by jeana-redhat — COMMENTED** [review]
One totally ignorable suggestion, LGTM

**Pull request merged**